examples/outsim-advanced-ts/.env

```diff
@@ -0,0 +1,4 @@
+DEBUG=node-insim-outsim-basic-ts
+HOST=127.0.0.1
+PORT=29999
```

Comment:
The default OutSim port should be 29997 like in the other OutSim example, also because 29999 is used by InSim by default.